db/dcrpg/go.mod

```diff
@@ -31,4 +31,6 @@ require (
 replace (
 	github.com/decred/dcrdata/api/types => ../../api/types
 	github.com/decred/dcrdata/db/cache => ../cache
+	github.com/decred/dcrdata/db/dbtypes => ../dbtypes
```

Comment:
Was getting a linter warning about some new fields in `dbtypes.PoolTicketsData`.